README.md

```diff
@@ -17,7 +17,7 @@ placed anywhere on the page. Inside the <head> tag or immediately after the .pro
 
```

Comment:
We can also include render method, with similar docs to https://docs.hcaptcha.com/configuration#explicitly-render-hcaptcha